monkey/agent_plugins/credentials_collectors/chrome/src/chrome_browser_local_data.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path, PurePath
+from typing import Collection, Iterator, Optional
+
+from .chrome_browser_local_state import ChromeBrowserLocalState
+
+
+class ChromeBrowserLocalData:
```

Comment:
NIT: I'd recommend making these functions read-only properties instead. Since the object is called `ChromeBrowserLocalData`, the interface makes more sense if I ask for `ChromeBrowserLocalData.profile_names` instead of `ChromeBrowserLocalData.get_profile_names()`.